### PR TITLE
Lovely LIDAR Library

### PR DESCRIPTION
**Pull request by ladyada**

### CODE REVIEW

**Comment by ladyada** [comment]
travis passed on first PR, i get a trophy or something!? 😆

**Comment by sommersoft** [comment]
> travis passed on first PR, i get a trophy or something!? 😆

I'd say that's deserving of a :taco:. Actually, make it two! :taco:

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.todo',
+]
+
+# TODO: Please Read!
+# Uncomment the below if you use native CircuitPython modules such as
+# digitalio, micropython and busio. List the modules you use. Without it, the
+# autodoc module docs will fail to generate with a warning.
+# autodoc_mock_imports = ["digitalio", "busio"]
```
Comment (sommersoft) [inline docs/conf.py:23]: Surprised Shpinx passed without mocking `digitalio` and `micropython`. :man_shrugging:
 Everything else looks good to me, but I'll leave this as a comment and not a review, for now.

Comment (kattni) [reply docs/conf.py:23]: I too am surprised by this. I thought Travis didn't pay attention to `requirements.txt` until we had another step done, but apparently it does.

**Review by kattni — APPROVED** [review]
🌮s all around! Looks good!

File: docs/conf.py (hunk 1 above)
Comment (ladyada) [reply docs/conf.py:23]: oops i dont know. we remove that for pypi anyways so it would be removed anyhow?

**Comment by ladyada** [comment]
huzzah!

File: docs/conf.py (hunk 1 above)
Comment (tannewt) [reply docs/conf.py:23]: `digitalio` and `micropython` are brought in via Blinka and the updated requirements.txt. Good job all! 🌮

**Pull request merged**